ui/app/templates/components/dashboard/vault-configuration-details-card.hbs

```diff
@@ -0,0 +1,38 @@
+<h3 class="title is-4" data-test-configuration-details-title>Configuration details</h3>
+
+<Hds::Table @caption="Vault configuration details" class="is-border-spacing-revert">
+  <:body as |B|>
+    <B.Tr>
+      <B.Td>API_ADDR</B.Td>
+      <B.Td data-test-vault-config-details="api_addr">{{@vaultConfiguration.api_addr}}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>Default lease TTL</B.Td>
+      <B.Td data-test-vault-config-details="default_lease_ttl">{{format-duration
+          @vaultConfiguration.default_lease_ttl
+        }}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>Max lease TTL</B.Td>
+      <B.Td data-test-vault-config-details="max_lease_ttl">{{format-duration @vaultConfiguration.max_lease_ttl}}</B.Td>
+    </B.Tr>
+    <B.Tr>
+      <B.Td>TLS</B.Td>
```

Comment:
This might be a question for designed but this should probably read `TLS disabled` since `TLS` doesn't give much information on its own

Reply:
Looking at designs it looks like the text should actually read "Enabled" or "Disabled"